interface/wx/webview.h

```diff
@@ -459,12 +459,50 @@ class wxWebView : public wxControl
     */
     virtual void Reload(wxWebViewReloadFlags flags = wxWEBVIEW_RELOAD_DEFAULT) = 0;
 
-    /**
-        Runs the given javascript code.
-        @note When using wxWEBVIEW_BACKEND_IE you must wait for the current
+	/**
+        Gets current IE emulation level for the current program.
```

Comment:
For consistency with the other methods, there should be a blank line between the summary and the rest of the documentation (here and below).